Backend/app/rider-platform/rider-app/Main/src/API/UI/Search.hs

```diff
@@ -70,6 +71,7 @@ type API =
     :> ReqBody '[JSON] SearchReq
     :> Servant.Header "x-bundle-version" Version
     :> Servant.Header "x-client-version" Version
+    :> Header "device" Text
```

Comment:
why header?

Reply:
@hkmangla fronted is sending device in header

Reply:
rename it `x-device`